auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -274,45 +268,6 @@ public AuthenticationRequest loginWithOTP(@NonNull String mfaToken, @NonNull Str
         return loginWithToken(parameters);
     }
 
-    /**
-     * Log in a user with a OAuth 'access_token' of a Identity Provider like Facebook or Twitter using <a href="https://auth0.com/docs/api/authentication#social-with-provider-s-access-token">'\oauth\access_token' endpoint</a>
-     * The default scope used is 'openid'.
-     * Example usage:
-     * <pre>
-     * {@code
-     * client.loginWithOAuthAccessToken("{token}", "{connection name}")
-     *      .start(new BaseCallback<Credentials>() {
-     *          {@literal}Override
-     *          public void onSuccess(Credentials payload) { }
-     *
-     *          {@literal}Override
-     *          public void onFailure(AuthenticationException error) { }
-     *      });
-     * }
-     * </pre>
-     *
-     * @param token      obtained from the IdP
-     * @param connection that will be used to authenticate the user, e.g. 'facebook'
-     * @return a request to configure and start that will yield {@link Credentials}
-     */
-    @NonNull
```

Comment:
Are you planning to create a PR for v1 to deprecate this, and all other public methods removed that will be removed?

Reply:
Yes, I'm tracking the deprecation here SDK-2206